index.html

```diff
@@ -183,12 +180,17 @@ <h2>Runs inside your cluster</h2>
         <div class="medium-6 columns">
           <aside class="cli">
             <pre>
-<em>// install brigade using helm</em>
-helm repo add brigade https://brigadecore.github.io/charts
-helm install brigade/brigade --name brigade-server
+<em>// install Brigade using Helm v3.7.0+</em>
+export HELM_EXPERIMENTAL_OCI=1
 
-<em>// check brigade status</em>
-helm get brigade-server
+helm install brigade \
+oci://ghcr.io/brigadecore/brigade \
+--version v2.0.0 \
+--create-namespace \
+--namespace brigade
+
+<em>// check Brigade status</em>
+helm get all brigade --namespace brigade
```

Comment:
All of this renders oddly because there's a background element that looks like a terminal windows and was perfectly sized for the old text. The new text overflows the terminal window, which looks strange in itself, but the overflow also contrasts poorly with the background.

Reply:
Yea, may need help from @flynnduism here.

Reply:
Sounds like the direction we're moving in is to merge these content changes to a new v2 branch and @flynnduism will also target that branch with style changes.

Based on that, I consider this resolved.